client/app/components/sections/ManageAvailability/ManageAvailability.js

```diff
@@ -70,6 +81,15 @@ class ManageAvailability extends Component {
     loadInitialDataIfNeeded(this.props);
   }
 
+  componentWillUpdate(nextProps) {
+    // manage location hash
+    if (nextProps.isOpen && window.location.hash !== '#manage-availability') {
```

Comment:
Will the `componentWillUpdate` function be run in the server rendering mode? If yes, then I think the `window.location.hash` will throw a null pointer. Should this be wrapped in `canUseDom`?